package.json

```diff
@@ -76,27 +69,38 @@
     "redux-debounce": "1.0.1",
     "redux-thunk": "2.3.0",
     "taskcluster-client-web": "8.0.4",
-    "taskcluster-lib-scopes": "10.0.1"
+    "taskcluster-lib-scopes": "10.0.1",
+    "webpack": "4.23.1",
+    "webpack-cli": "3.1.2"
   },
   "devDependencies": {
+    "@neutrinojs/eslint": "9.0.0-beta.1",
     "angular-mocks": "1.7.5",
     "enzyme": "3.7.0",
     "enzyme-adapter-react-16": "1.6.0",
+    "eslint": "5.8.0",
+    "eslint-config-airbnb": "17.1.0",
+    "eslint-plugin-import": "2.14.0",
+    "eslint-plugin-jsx-a11y": "6.1.2",
+    "eslint-plugin-react": "7.11.1",
     "fetch-mock": "7.2.2",
     "jasmine-core": "3.3.0",
     "jasmine-jquery": "2.1.1",
-    "karma": "1.7.1",
+    "karma": "3.1.1",
     "karma-firefox-launcher": "1.1.0",
     "karma-jasmine": "1.1.2",
-    "neutrino-preset-karma": "4.2.1"
+    "karma-webpack": "4.0.0-beta.0",
+    "webpack-dev-server": "3.1.10"
   },
   "scripts": {
-    "build": "node ./node_modules/neutrino/bin/neutrino build --presets ./neutrino-custom/production.js",
-    "lint": "node ./node_modules/eslint/bin/eslint.js --cache --ext js,jsx \".*.js\" \"*.js\" ui/ tests/ui/ neutrino-custom/lint.js",
-    "start": "node ./node_modules/neutrino/bin/neutrino start --presets ./neutrino-custom/development.js",
-    "start:local": "BACKEND_DOMAIN=http://localhost:8000 node ./node_modules/neutrino/bin/neutrino start --presets ./neutrino-custom/development.js",
-    "start:stage": "BACKEND_DOMAIN=https://treeherder.allizom.org node ./node_modules/neutrino/bin/neutrino start --presets ./neutrino-custom/development.js",
-    "test": "node ./node_modules/neutrino/bin/neutrino test --presets ./neutrino-custom/test.js",
-    "test:watch": "node ./node_modules/neutrino/bin/neutrino test --watch --presets ./neutrino-custom/test.js"
+    "build": "node ./node_modules/webpack/bin/webpack.js --mode production",
+    "build:dev": "node ./node_modules/webpack/bin/webpack.js --mode development",
```

Comment:
I've added a new `yarn build:dev` which allows creating a quicker build if wanting to eg run Selenium tests locally. It will take ~half the time of the standard (production) `yarn build`.

Reply:
oh nice!  That'll be really helpful.  :)